kumascript/index.js

```diff
@@ -50,7 +51,8 @@ const renderFromURL = async (
         `Tried to find a folder called ${Document.urlToFolderPath(url)}`
     );
   }
-  const { rawHTML, metadata, fileInfo } = document;
+  const { rawBody, metadata, fileInfo, isMarkdown } = document;
+  const rawHTML = isMarkdown ? markdownToHTML(rawBody) : rawBody;
```

Comment:
I didn't manage to fully review as I was reading up on unify things. One thing I wanted to feedback: @peterbe If I see this right `renderFromURL` is the place in a _normal_ build where we do the conversion. I guess this is a rather central place (except for `getPage`) however, I would argue that moving that step out of here and make it more explicit (with a potential sacrifice of doing it in more than one spot) might be beneficial. Otherwise, nothing that caught my eye so far 👍

Reply:
I'm not sure what your question is but my logic is that I didn't want to do it in `Document.read()` because there are many places where it uses `Document.read()` to just see if the document exists, without actually needing to render it. So it'd be a waste to do the markdown conversion it's just going to get kicked out sooner or later by the LRU.
For example, the `smartLink()` and the `broken_link` flaw checker will see if the URL can be traced to a document. 

But that brings up an optimization idea. Perhaps can put it back into `Document.read()` but using `Document.read()` should be discouraged if you aren't going to actually build it. 
For example, the `smartLink()` function could do something like:

```diff
  smartLink(href, title, content, subpath, basepath) {
    let flaw;
    let flawAttribute = "";
-   const page = this.info.getPageByURL(href);
+   const page = this.info.getPageByURLLite(href);

```
(terrible name by the way!) 
There's a LOT going on in `Document.read()` but basically all you need is to find out if it can be resolved and if so, what is its `title` or something. 

As a matter of fact, I looked through `kumascript/src/api.js` and `build/flaws.js` where it does see if a URL does map to a document on disk and in both cases it just does two things:

1. Does it exist?
2. If so, what is the `.url`

Premature optimizations are dangerous but it might be worth it so doesn't take 1h20m to make a complete build :)

Another thing that I know this PR doesn't solve is that it might potentially swallow and hide is if the markdown conversion fails. I think it needs to be solved as a follow-up to this PR but if you have some completely corrupt and busted Markdown in some `index.md` file, it should fail in a decent way. It shouldn't be treated as a flaw but I'm just not sure what would happen.

Reply:
I did some research into this. The optimization isn't just premature, it's pointless too. `Document.read()` is very fast. 
I can read ~12k URLs in 3.1 seconds (eyeballs average after 5 runs). It takes 2.1 seconds if I remove that `memoize()` wrapper because I think it's adding more overheads and LRU evictions than just re-reading. 

I don't know if it's worth worrying about `Document.read()` being slow. 
I still think it makes sense to not do the markdown conversion within, but doing it in kumascript/src/ isn't great either.